app/views/user_mailer/delete_work_notification.html.erb

```diff
@@ -1,9 +1,11 @@
 <% content_for :message do %>
-  <p>Dear <%= @user.default_pseud.byline %>,</p>
-
-  <p>Your work <i><%= style_bold(@work.title.html_safe) %></i> was deleted at <%= (@work.pseuds.count > 1 && @user != User.current_user) ? (ts("the request of ") + style_pseud_link(User.current_user.default_pseud)).html_safe : "your request" %>.</p>
-  
-  <p>If you have questions, please <%= support_link("contact Support") %>.</p>  
-  
-  <p>Attached is a copy of your work for your reference.</p>
+<p><%= t('.hello', user:  @user.default_pseud.byline) %></p>
+<p><%= (@work.pseuds.count > 1 && @user != User.current_user) ?
```

Comment:
Would it be possible to do something like `<% pretty_title = style_bold(@work.title.html_safe) %>` so you could have just `title: <i>#{pretty_title}</i>.html_safe`? (Just trying to see if there's a way to make it easier to read.)